File: Application/VirgilCypto/ViewController.m
//
//  ViewController.m
//  VirgilCypto
//
//  Created by Pavel Gorb on 9/23/15.
//  Copyright (c) 2015 VirgilSecurity. All rights reserved.
//

#import "ViewController.h"
#import "VSCVirgilVersion.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    VSCVirgilVersion *version = [[VSCVirgilVersion alloc] init];
    NSLog(@"Virgil Version: %@", [version versionString]);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

@end

File: Application/VirgilCryptoSwift/ViewController.swift
//
//  ViewController.swift
//  VirgilCryptoSwift
//
//  Created by Pavel Gorb on 9/23/15.
//  Copyright (c) 2015 VirgilSecurity. All rights reserved.
//

import UIKit
import Foundation

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let version = VSCVirgilVersion()
        print("Virgil Version: \(version.versionString())")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

